### PR TITLE
refactoring db.py, for *messages and *rooms plus Docstrings for both

### PR DESCRIPTION
**Pull request by Murithijoshua**
* created a new helper function to reduce redundant variables
* worked on docstrings

### CODE REVIEW

**Review by man-of-mind — APPROVED** [review]

**Review by o4codes — CHANGES_REQUESTED** [review]
this is great work,
I'd advise the docstring for each function uses google standard as it has been adopted to be used in this project
https://google.github.io/styleguide/pyguide.html
The guideline for doc string can be found under section 2.4.4

**Comment by Murithijoshua** [comment]
Thanks for the feedback, 
I will be rolling out new changes.

**Review by o4codes — APPROVED** [review]

**Comment by o4codes** [comment]
@blacdev 
I think this is alright to be merged

File: backend/db.py [hunk 1]
```diff
@@ -198,61 +198,87 @@ def delete_file(self, file_url):
...
-    Args:
-        user_id (str): The user id
-
-    Returns:
-        [List]: [description]
+def getOrg(org: str):
+    """ This is a helper function
+    :param org: organizational id
+    :type: string
+    :rtype: class object referrencing DataStorage
```
[the remark this answers is not in the document]
Comment (Murithijoshua) [reply backend/db.py]: this is resolved

**Review by zxenonx — APPROVED** [review]

**Review by zxenonx — APPROVED** [review]

**Pull request merged**